### PR TITLE
feat: add GereralContentPage

### PR DESCRIPTION
**Pull request by jendiamond**
Connected to [APPS-1886](https://jira.library.ucla.edu/browse/APPS-1886)

### CODE REVIEW

**Review by pghorpade — APPROVED** [review]
👍

File: pages/_parent/_slug.vue [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<template lang="html">
+    <main
+        id="main"
+        class="page page-general-content"
+    >
+        <nav-breadcrumb
+            :title="page.title"
+            :is-meap="true"
```
Comment (aprigge) [inline pages/_parent/_slug.vue:8]: Remove `is-meap=true`

File: pages/_parent/index.vue [hunk 2]
```diff
@@ -0,0 +1,80 @@
+<template lang="html">
+    <main
+        id="main"
+        class="page page-general-content"
+    >
+        <nav-breadcrumb
+            :title="page.title"
+            :is-meap="true"
```
Comment (aprigge) [inline pages/_parent/index.vue:8]: Remove `is-meap=true`

**Review by matheusgomes062 — APPROVED** [review]
:+1:

**Pull request merged**